Makefile

```diff
@@ -78,6 +78,7 @@ test-long: clean fmt REBUILD
 	go test -v -race -tags='testing debug' -timeout=360s $(pkgs)
 bench: clean fmt REBUILD
 	go test -tags='testing' -timeout=120s -run=XXX -bench=. $(pkgs)
+	go test -race -tags='testing debug' -timeout=120s -run=XXX -bench=. $(pkgs)
```

Comment:
It does both now.

I can't remember if I intended to commit this or not.

Basically, the tests were running _really_ slow for a bit, and I was trying to figure out what was causing the extreme slowdown.

Turned out that both the wallet and the boltdb changes had each slowed the project down by a few hundred milliseconds per tester object created. I can remove it now that the tests are running reasonably quickly again.